### PR TITLE
feat(language): Handle languages other than English

### LINKED ISSUE #7192: Languages other than English
CAPI supports languages other than english, e.g. [Spanish](http://content.guardianapis.com/search?api-key=test&lang=es) and [Japanese](http://content.guardianapis.com/search?api-key=test&lang=ja)

Screen readers should be informed that part of the page’s content is in a different language, using the [`lang` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang)

### PR DESCRIPTION
**Pull request by georgeblahblah**
## What does this change?

- Adds two fields, `lang` and `isRightToLeftLang` to `FEArticleType`
- Parses `lang` and `isRightToLeftLang` in `ArticleBody`
  - if `isRightToLeftLang` is `true`, the `dir` attribute on `#maincontent` should be `rtl`. Otherwise it can be `undefined`
  - **Note**: if `lang` is `en`:
    - I suggest we should ignore it. This is because CAPI defaults to `en` for languages it doesn't recognise. By passing this value through, we could potentially set an incorrect value. This is how [Frontend currently works](https://github.com/guardian/frontend/blob/main/common/app/views/fragments/langAttributes.scala.html#L2). See https://github.com/guardian/content-api/blob/main/porter/src/main/scala/com.gu.contentapi.porter/integration/LanguageDetector.scala#L26
    - otherwise, pass through the value received from Frontend
- Passes `lang` and `isRightToLeftLang` to `ArticleBody` from the Layouts

## Why?

- To resolve #7192

### CODE REVIEW

File: dotcom-rendering/src/web/components/ArticleBody.tsx [hunk 1]
```diff
@@ -114,6 +116,16 @@ const globalLinkStyles = (palette: Palette) => css`
...
 
+// CAPI only supports certain languages. If CAPI doesn't recognise the language,
+// it defaults to `en`. We should filter out `en` so we don't set an incorrect value.
+// See https://github.com/guardian/content-api/blob/main/porter/src/main/scala/com.gu.contentapi.porter/integration/LanguageDetector.scala#L17
+const decideLanguage = (language = ''): string | undefined =>
+	language != 'en' ? language : undefined;
+
+const decideLanguageDirection = (
+	isRightToLeftLang = false,
+): 'rtl' | undefined => (isRightToLeftLang ? 'rtl' : undefined);
```
Comment (AshCorr) [inline dotcom-rendering/src/web/components/ArticleBody.tsx:127]:
```suggestion
): 'rtl' | undefined => isRightToLeftLang ? 'rtl' : undefined;
```

**Review by AshCorr — APPROVED** [review]

**Review by mxdvl — APPROVED** [review]
Ça me semble bien.

**Review by bryophyta — APPROVED** [review]
this looks great!

File: dotcom-rendering/src/web/components/ArticleBody.tsx (hunk 1 above)
Comment (georgeblahblah) [reply dotcom-rendering/src/web/components/ArticleBody.tsx:127]: I thought so too! but `prettier` [adds them back in](https://github.com/guardian/dotcom-rendering/actions/runs/4223115519/jobs/7332452396#step:5:10)

**Pull request merged**